src/main/java/net/snowflake/client/core/HttpUtil.java

```diff
@@ -318,8 +344,15 @@ public static CloseableHttpClient buildHttpClient(
         logger.error(errors.toString(), true);
         throw new RuntimeException(err); // rethrow the exception
       }
+    } else if (key != null) {
+      logger.debug("Omitting trust manager instantiation as OCSP mode is set to {}", key.getOcspMode());
+    } else {
+      logger.debug("Omitting trust manager instantiation as configuration is null");
```

Comment:
do we want to say `is null` or `is not provided`?

Reply:
Changed to `is not provided`. Thanks.